data/hackathon/scenario7.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Accelerate GenAI experimentation by using an open source Enterprise-grade tool and software supply chain to get started
+exercise: 7
+date: '2024-06-25'
+tags: ['podman desktop','ai lab','genai', 'granite','ai applications']
+draft: false
+authors: ['default']
+summary: "Demonstrate Podman Desktops in-built"
+---
+
+ACME loved the flexibility at enterprise grade quality and scale of OpenShift AI and also the game changing approach to LLMs through InstructLab. 
+But all this excitement needs to be pointed in to the right direction with the right tooling right from when experimentation starts.
+"Can you help us?" they want to know - "I am glad you asked" is what you are saying.
+
+In this scenario you showcase the Podman tooling to get started while having Enterprise grade deployments as the destination in mind.
+1) Get Podman Desktop running on  your laptop.
+2) Install the AI Lab extension
+3) Use the AI Lab extension to build your demo app and to access to your code
+4) Utilize the Recipe catalogue to deploy the Summarizer recipe
+5) Change the application code so that it says 'Hello from Team <Your team name>!'
+6) Take a pdf document and let the Summarizer app summarize it.
+7) Take a screenshot and show that you changed the Greeting as well as that the summarizer app ran successfully.
+
+## 7.0 - Be in the Know...
+Documentation you might find useful
+- https://podman-desktop.io/extensions/ai-lab
+- https://developers.redhat.com/articles/2024/05/07/podman-ai-lab-getting-started?source=sso#catalog_of_open_source_models
+
+## 7.1 - Setting Up
+The ACMEFS team wants to see how you install Podman Desktop and the AI Lab extension on your laptop
+- Demonstrate the installation process or upgrade process of Podman Desktop.
+
+## 7.2 - Introducing Podman
+Explain what you see when going through podman desktop.
+- The team wants to know what version of Podman is required to install the AILab extension.
+- Show the ACME team how you can view and access containers, images and volumes
+- Browse the current models installed and show how you can download a new model.
```

Comment:
Let's include a screenshot here of Podman AI to make the page a little more interesting.